test/feeds.test.js

```diff
@@ -133,6 +134,83 @@ describe('test POST /feeds endpoint', () => {
   });
 });
 
+describe('test DELETE /feeds/cache endpoint', () => {
+  it('should respond with a 403 status trying to clear cache without logging in', async () => {
+    const res = await request(app).delete('/feeds/cache').send().set('Accept', 'application/json');
+    expect(res.status).toEqual(403);
+  });
+
+  it('should respond with a 403 status trying to clear cache when logged in as normal user', async () => {
+    login('Johannes Kepler', 'user1@example.com');
+    const res = await request(app).delete('/feeds/cache').send().set('Accept', 'application/json');
+    expect(res.status).toEqual(403);
+  });
+
+  it('should respond with a 204 status trying to clear cache when logged in as admin user', async () => {
+    loginAdmin('Johannes Kepler', 'user1@example.com');
+    const res = await request(app).delete('/feeds/cache').send().set('Accept', 'application/json');
+    expect(res.status).toEqual(204);
+  });
+
+  it('should respond with a 204 and all existing feed etag + lastModified be null', async () => {
+    const urlToId = (url) => hash(normalizeUrl(url));
+
+    const data = {
+      author: 'Post Author',
+      url: 'https://user.feed.com/feed.rss',
+      user: 'user',
+      id: urlToId('https://user.feed.com/feed.rss'),
+      link: 'https://user.feed.com/',
+      etag: 'etag',
+      lastModified: 'lastModified',
+    };
+
+    const data2 = {
+      author: 'Post Author2',
+      url: 'https://user2.feed.com/feed.rss',
+      user: 'user2',
+      id: urlToId('https://user2.feed.com/feed.rss'),
```

Comment:
These generated `id` properties (here and below on 182) are never read, so I wouldn't bother generating them, or importing `normalize-url` or hashing them above.  What you're doing is correct, but `Feed.create()` will do this for you, and you're not testing that it works, so don't bother doing its work for it.

Reply:
Totally removed the id property from `data`, `data2`, `data3`